app/services/create_proposed_organisation_edit.rb

```diff
@@ -36,6 +36,7 @@ def merge_in_non_published_fields
 
   def send_email_to_superadmin_about_org_edit
     superadmin_emails = user_klass.superadmins.pluck(:email)
```

Comment:
@decareano I think you've misunderstood me.  I was saying that we need to change the `send_email_to_superadmin_about_org_edit` method to use the `User.superadmin_emails` method, i.e.

```rb
  def send_email_to_superadmin_about_org_edit
    superadmin_emails = user_klass.superadmin_emails
    mailing = mailer_klass.edit_org_waiting_for_approval(organisation, superadmin_emails)
    mailing.deliver_now
  end
```

regarding the cukes, we are adjusting here a private method on the service, and adding a convenience method to the User class.  I don't think there is any need for the cukes to change, because we are not changing any functionality.  Arguably we should add a new spec for the User class to wrap the convenience method.

Also what will happen is we'll need to adjust the service spec.  The whole refactoring is actually motivated by these lines in the `create_proposed_organisation_edit_spec`:

```rb
      allow(user_klass).to receive_message_chain(:superadmins, :pluck).and_return(:admins)
      allow(mailer_class).to receive_message_chain :edit_org_waiting_for_approval, :deliver_now
```

that we have to mock out `receive_message_chain` indicates that our service is rather tightly bound to it's collaborators.  Having extracted the Rails pluck operation into the User class we could then change at least one of these mocks like so:

```rb
      allow(user_klass).to receive(:superadmin_emails).and_return(:admins)
```

Code climate is the main tool we're using for analysis since it combines all others - see below the CodeClimate is telling us that this PR fixes one code issue